packages/editor/src/components/mobile/bottom-sheet/cell.native.js

```diff
@@ -7,85 +7,132 @@ import { TouchableOpacity, Text, View, TextInput, I18nManager } from 'react-nati
  * WordPress dependencies
  */
 import { Dashicon } from '@wordpress/components';
+import { Component } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import styles from './styles.scss';
+import platformStyles from './cellStyles';
 
-export default function Cell( props ) {
-	const {
-		onPress,
-		label,
-		value,
-		valuePlaceholder = '',
-		drawSeparator = true,
-		icon,
-		labelStyle = {},
-		valueStyle = {},
-		onChangeValue,
-		children,
-		editable = true,
-		...valueProps
-	} = props;
+export default class Cell extends Component {
+	constructor() {
+		super( ...arguments );
+		this._valueTextInput = undefined;
```

Comment:
Maybe if we already know that this. _valueTextInput is undefined by default, we don't need to assign it to undefined?